contracts/SecurityTokenRegistry.sol

```diff
@@ -81,7 +81,7 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, Util, Pausable, Regist
         require(_owner != address(0));
         require(!(ITickerRegistry(tickerRegistry).isReserved(symbol, _owner, _name, _swarmHash)), "Trying to use non-valid symbol");
         symbols[symbol] = _securityToken;
-        securityTokens[_securityToken] = SecurityTokenData(symbol, _tokenDetails);
+        securityTokens[_securityToken] = SecurityTokenData(symbol, _tokenDetails, now);
         emit LogAddCustomSecurityToken(_name, symbol, _securityToken, now);
```

Comment:
same here, instead of `now` that should be an extra parameter being passed so we can migrate the reg date